roles/tests/test_api.py

```diff
@@ -76,6 +79,60 @@ def test_roles_init_new_repo_2(self):
             sorted(RepoPermission.author_permissions())
         )
 
+    def test_roles_init_new_repo_3(self):
```

Comment:
It is already a bit late, but instead of using numbers for all of these, it would be nice if you described what was under test.  So that if they fail it is easier to understand what broke. If Pylint complains about the method length, feel free to disable